src/CMakeLists.txt

```diff
@@ -27,7 +27,7 @@ if (MOVIES)
     )
 endif ()
 
-if ("${GTK_VERSION}" VERSION_LESS 3.18 OR "${VALA_VERSION}" VERSION_LESS 0.30)
+if ("${VALA_VERSION}" VERSION_LESS 0.30)
     message(WARNING "Using legacy GTK API")
```

Comment:
I think we can ship this completely (even in code) because we rely on GTK 3.20 and that legacy code path can't be used.